src/dictionary.ts

```diff
@@ -918,12 +948,12 @@ export const en_dk : Lexeme[] = [
   },
   {
     word: "rename",
-    translations: ["omnavngive"],
+    translations: ["omnavngive", "omdøb"],

```

Comment:
Maybe *omdøb* should be the first suggestion, i.e. before *omnavngive*? Is *omnavngive* even a word?

Reply:
Quick Google suggests it's being used in articles but I don't see it in the dictionaries.